app/controllers/newbookings_controller.rb

```diff
@@ -40,6 +45,14 @@ def destroy
     redirect_to newbookings_index_path
   end
 
+  def sort_by_date
+    @newbooking = @newbooking.sort_by(&:date)
```

Comment:
This is really good for sorting locally. When you have a (small) number of values or already have them all in memory, this works very well.

Where this has some challenges is when you have lots of records and pagination. If I want to show the only the first 10 (of 500) records and I want to sort it by date, can you guess what would happen here?